department-of-veterans-affairs/va.gov-cms
# VACMS-5911: Add vamc custom module and op status ief banner auto governance

## Description

Closes #5911

## Testing done
Visual / behat

## QA steps
1. Go node/2370/edit
2. Create a banner alert with IEF
3. Save.
4. Find the node from /admin/content
5. See that it's assigned to Wilmington section via field_administration

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
